src/components/Api.js

```diff
@@ -81,12 +78,7 @@ export default class Api {
         name: name,
         link: link,
       }),
-    }).then((res) => {
-      if (res.ok) {
-        return res.json();
-      }
-      return new Promise.reject(`Error : ${res.status}`);
-    });
+    }).then(this._checkResponse);
   }
 
   confirmDeleteButton(cardID) {
```

Comment:
* You haven't corrected something https://github.com/szlaurel/se_project_aroundtheus/pull/1#discussion_r1283759502 


```js

  confirmDeleteButton(cardID) {
    return fetch(`${this._baseUrl}/cards/${cardID}`, {
      method: "DELETE",
      headers: this._headers,
    }).then(this._checkResponse);
  }

```